src/app/components/pages/IsaacCompetition/Accordion/AccordionItem.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { memo } from "react";
+import { Card, CardBody, CardHeader, Collapse } from "reactstrap";
+
+interface AccordionItemProps {
+  id: string;
+  title: string;
+  section: string[];
+  open: string | null;
+  setOpenState: (id?: string) => void;
+}
+
+const renderSectionContent = (section: string[]) => (
+  <>
+    {section[0]}
+    <ul>
+      {section.slice(1, -1).map((item) => (
+        <li key={item}>{item}</li>
+      ))}
+    </ul>
+    {section[section.length - 1]}
```

Comment:
i ponder how this could be done a bit more intuitively - maybe we have a nested list that can signify to the code that it's a list? This doesn't feel very scalable for Accordions in the long run to expect the first sentence to be so, then an unordered list, then a final sentence.